src/components/UiNotificationBadge/UiNotificationBadge.tsx

```diff
@@ -1,95 +1,60 @@
-	[EColors.PRIMARY_700]: "bg-primary-700",
-	[EColors.PRIMARY_800]: "bg-primary-800",
-	[EColors.PRIMARY_900]: "bg-primary-900",
+};
 
-	[EColors.PRIMARY_ALT]: "bg-primary-alt",
-	[EColors.PRIMARY_ALT_300]: "bg-primary-alt-300",
-	[EColors.PRIMARY_ALT_400]: "bg-primary-alt-400",
-	[EColors.PRIMARY_ALT_500]: "bg-primary-alt-500",
-	[EColors.PRIMARY_ALT_600]: "bg-primary-alt-600",
-	[EColors.PRIMARY_ALT_700]: "bg-primary-alt-700",
+const offsetClasses = {
+	[EBadgeOffset.DEFAULT]: "-right-xxxs -top-xxxs",
+	[EBadgeOffset.OFFSET_BOTTOM_RIGHT]: "-right-xxxs -bottom-xxxs",
+	[EBadgeOffset.OFFSET_TOP_LEFT]: "-left-xxxs -top-xxxs",
+	[EBadgeOffset.OFFSET_BOTTOM_LEFT]: "-left-xxxs -bottom-xxxs",
+};
 
-	[EColors.WARNING]: "bg-warning",
-	[EColors.WARNING_300]: "bg-warning-300",
-	[EColors.WARNING_400]: "bg-warning-400",
-	[EColors.WARNING_500]: "bg-warning-500",
-	[EColors.WARNING_600]: "bg-warning-600",
-	[EColors.WARNING_700]: "bg-warning-700",
+const kindClasses = {
+	[EBadgeTypes.DEFAULT]: `
+		bg-error-500
 
-	[EColors.SECONDARY]: "bg-secondary",
-	[EColors.SECONDARY_300]: "bg-secondary-300",
-	[EColors.SECONDARY_400]: "bg-secondary-400",
-	[EColors.SECONDARY_500]: "bg-secondary-500",
-	[EColors.SECONDARY_600]: "bg-secondary-600",
-	[EColors.SECONDARY_700]: "bg-secondary-700",
+	`,
+	[EBadgeTypes.PRIMARY]: `
+		bg-primary-800
 
-	[EColors.SECONDARY_ALT]: "bg-secondary-alt",
-	[EColors.SECONDARY_ALT_200]: "bg-secondary-alt-200",
-	[EColors.SECONDARY_ALT_300]: "bg-secondary-alt-300",
-	[EColors.SECONDARY_ALT_400]: "bg-secondary-alt-400",
-	[EColors.SECONDARY_ALT_500]: "bg-secondary-alt-500",
-	[EColors.SECONDARY_ALT_600]: "bg-secondary-alt-600",
-	[EColors.SECONDARY_ALT_700]: "bg-secondary-alt-700",
+	`,
 
-	[EColors.ERROR]: "bg-error",
-	[EColors.ERROR_300]: "bg-error-300",
-	[EColors.ERROR_400]: "bg-error-400",
-	[EColors.ERROR_500]: "bg-error-500",
-	[EColors.ERROR_600]: "bg-error-600",
-	[EColors.ERROR_700]: "bg-error-700",
+};
 
-	[EColors.ACCENT]: "bg-accent",
-	[EColors.ACCENT_300]: "bg-accent-300",
-	[EColors.ACCENT_500]: "bg-accent-500",
-	[EColors.ACCENT_600]: "bg-accent-600",
-	[EColors.ACCENT_700]: "bg-accent-700",
+const sizeClasses = {
+	[EBadgeSizes.DEFAULT]: `
+		h-sm
+		min-w-sm
+		p-xxxs
+		text-xxs
+	`,
+	[EBadgeSizes.LARGE]: `
+		h-lg
+		min-w-lg
+		p-xxs
+		justify-center
+		text-2xl
+		border-2
 
-	[EColors.ACCENT_ALT]: "bg-accent-alt",
-	[EColors.ACCENT_ALT_300]: "bg-accent-alt-300",
-	[EColors.ACCENT_ALT_400]: "bg-accent-alt-400",
-	[EColors.ACCENT_ALT_500]: "bg-accent-alt-500",
-	[EColors.ACCENT_ALT_600]: "bg-accent-alt-600",
-	[EColors.ACCENT_ALT_700]: "bg-accent-alt-700",
-};
+	`
 
-const offsetClasses = {
-	[EBadgeOffset.DEFAULT]: "-right-xxxs -top-xxxs",
-	[EBadgeOffset.OFFSET_BOTTOM_RIGHT]: "-right-xxxs -bottom-xxxs",
-	[EBadgeOffset.OFFSET_TOP_LEFT]: "-left-xxxs -top-xxxs",
-	[EBadgeOffset.OFFSET_BOTTOM_LEFT]: "-left-xxxs -bottom-xxxs",
 };
 
 export const UiNotificationBadge: React.FC<UiNotificationBadge> = ({
 	offset,
```

Comment:
I think it's ok to set a default parameter here, also the badge should be shown even if there is no content in it - as if an empty space was passed now